src/modules/reporting/actions/estimate-submit-market-contribute.js

```diff
@@ -0,0 +1,28 @@
+import { augur } from 'services/augurjs'
+import logError from 'utils/log-error'
+import noop from 'utils/noop'
+
+export const estimateSubmitMarketContribute = (marketId, amount, callback = logError) => (dispatch, getState) => {
+  const { loginAccount, marketsData } = getState()
+
+  const market = marketsData[marketId]
```

Comment:
Should this make use of a selector? I'm not sure if the answer is yes but might be a worthwhile conversation. 

It would prevent this action from being directly dependent on the shape of the state.

Reply:
I see marketsData used in the code base, I think it's fine.